config/environments/test.rb

```diff
@@ -22,7 +22,7 @@
   config.public_file_server.headers = { "cache-control" => "public, max-age=3600" }
 
   # Show full error reports.
-  config.consider_all_requests_local = true
+  config.consider_all_requests_local = false
```

Comment:
I'm not sure about turning this off for all tests, as I sometimes use the traceback in the response in request specs for debugging tests. Can we turn if off for just the tests we need to have it off for?

Reply:
Yeh, can do that

Reply:
Hmm not sure this is the easier - we can send a header 'X-Requested-With': 'XMLHttpRequest' in the test to override it I think though, if that is preferable?

But is it not better to turn it off as then it's more representative of how the app will actually behave. To debug can't you just temporarily turn on logging or put a debug statement in `ErrorsController`?